packages/core/system/src/forward-ref/__tests__/forward-ref.test.tsx

```
@@ -0,0 +1,42 @@
+import { render, screen } from "@testing-library/react";
```

ESLint

[error] Unable to resolve path to module '@testing-library/react'.

(import-x/no-unresolved)